runtime/vm/ContinuationHelpers.cpp

```diff
@@ -536,6 +536,22 @@ acquireVThreadInspector(J9VMThread *currentThread, jobject thread, BOOLEAN spin)
 	threadObj = J9_JNI_UNWRAP_REFERENCE(thread);
 	vthreadInspectorCount = J9OBJECT_I64_LOAD(currentThread, threadObj, vm->virtualThreadInspectorCountOffset);
 	if (vthreadInspectorCount < 0) {
+		if (VM_VMHelpers::isThreadSuspended(currentThread, threadObj)) {
+			/* Thread is suspended, so we can directly allow access to thread without waiting for transition to complete.
+			 * Decrement the inspector count to represent inspector during suspended transition.
+			 */
+			if (!objectAccessBarrier.inlineMixedObjectCompareAndSwapU64(
+															currentThread,
+															threadObj,
+															vm->virtualThreadInspectorCountOffset,
+															(U_64)vthreadInspectorCount,
+															((U_64)(vthreadInspectorCount - 1)))
```

Comment:
The indentation is excessive here (two tabs more than line 543 is sufficient).
Also near lines 595 and 608.